enterprise/core-edge/src/main/java/org/neo4j/coreedge/discovery/procedures/AcquireEndpointsProcedure.java

```diff
@@ -36,8 +36,9 @@
 import org.neo4j.helpers.collection.Iterators;
 import org.neo4j.kernel.api.exceptions.ProcedureException;
 import org.neo4j.kernel.api.proc.CallableProcedure;
+import org.neo4j.kernel.api.proc.Context;
```

Comment:
Please also remove all the binary enterprise/backup/** files committed above (can't comment there so have to write this here)